### PR TITLE
Add Keyboard - Romeo

### PR DESCRIPTION
**Pull request by coseyfannitutti**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/coseyfannitutti/romeo/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ROMEO
+
+![romeo]()
```
Comment (zvecr) [inline keyboards/coseyfannitutti/romeo/readme.md]: Can we get an image here, or remove.

File: keyboards/coseyfannitutti/romeo/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+A staggered 40% that can be assembled with only through hole components, including usb type-c, and can also be completely covered with most GMK base kits
+
+Keyboard Maintainer: [coseyfannitutti](https://github.com/coseyfannitutti)  
+Hardware Supported: ROMEO, atmega328p  
+Hardware Availability: [cftkb.com](http://www.cftkb.com), [GitHub](https://github.com/coseyfannitutti/romeo)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make coseyfannitutti/discipad:default
```
Comment (zvecr) [inline keyboards/coseyfannitutti/romeo/readme.md]:
```suggestion
    make coseyfannitutti/romeo:default
```

File: keyboards/coseyfannitutti/romeo/romeo.c [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ */
+#include "romeo.h"
+
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+
+	matrix_init_user();
+}
```
Comment (zvecr) [inline keyboards/coseyfannitutti/romeo/romeo.c]:
```suggestion

```

File: keyboards/coseyfannitutti/romeo/readme.md (hunk 1 above)
Comment (coseyfannitutti) [reply keyboards/coseyfannitutti/romeo/readme.md]: Removed.

File: keyboards/coseyfannitutti/romeo/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_all(
+      KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_BSPC, 
+      KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_QUOT,
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/romeo/keymaps/default/keymap.c]:
```suggestion
      KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_QUOT,
```

File: keyboards/coseyfannitutti/romeo/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_all(
+      KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_BSPC, 
+      KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_QUOT,            
+      KC_LSFT, KC_SLSH, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_RSFT,
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/romeo/keymaps/default/keymap.c]:
```suggestion
      KC_LSFT, KC_SLSH, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_RSFT,
```

File: keyboards/coseyfannitutti/romeo/readme.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# ROMEO
+
+A staggered 40% that can be assembled with only through hole components, including usb type-c, and can also be completely covered with most GMK base kits
+
+Keyboard Maintainer: [coseyfannitutti](https://github.com/coseyfannitutti)  
+Hardware Supported: ROMEO, atmega328p  
+Hardware Availability: [cftkb.com](http://www.cftkb.com), [GitHub](https://github.com/coseyfannitutti/romeo)
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/romeo/readme.md]:
```suggestion
* Keyboard Maintainer: [coseyfannitutti](https://github.com/coseyfannitutti)
* Hardware Supported: ROMEO, ATmega328P
* Hardware Availability: [cftkb.com](http://www.cftkb.com), [GitHub](https://github.com/coseyfannitutti/romeo)
```

File: keyboards/coseyfannitutti/romeo/rules.mk [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/romeo/rules.mk]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
UNICODE_ENABLE = no         # Unicode
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output on port C6
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```

File: keyboards/coseyfannitutti/romeo/usbconfig.h [hunk 8]
```diff
@@ -0,0 +1,397 @@
...
+ * scope of the vendor ID. If you have registered your own VID with usb.org
+ * or if you have licensed a PID from somebody else, define it here. Otherwise
+ * you may use one of obdev's free shared VID/PID pairs. See the file
+ * USB-IDs-for-free.txt for details!
+ * *** IMPORTANT NOTE ***
+ * This template uses obdev's shared VID/PID pair for Vendor Class devices
+ * with libusb: 0x16c0/0x5dc.  Use this VID/PID pair ONLY if you understand
+ * the implications!
+ */
+#define USB_CFG_DEVICE_VERSION  0x01, 0x00
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/romeo/usbconfig.h]:
```suggestion
#define USB_CFG_DEVICE_VERSION  (DEVICE_VER & 0xFF), ((DEVICE_VER >> 8) & 0xFF)
```

File: keyboards/coseyfannitutti/romeo/usbconfig.h [hunk 9]
```diff
@@ -0,0 +1,397 @@
...
+#define USB_CFG_IS_SELF_POWERED         0
+/* Define this to 1 if the device has its own power supply. Set it to 0 if the
+ * device is powered from the USB bus.
+ */
+// max power draw with maxed white underglow measured at 120 mA (peaks)
+#define USB_CFG_MAX_BUS_POWER           100
+/* Set this variable to the maximum USB bus power consumption of your device.
+ * The value is in milliamperes. [It will be divided by two since USB
+ * communicates power requirements in units of 2 mA.]
+ */
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/romeo/usbconfig.h]:
```suggestion
```

File: keyboards/coseyfannitutti/romeo/usbconfig.h [hunk 10]
```diff
@@ -0,0 +1,397 @@
...
+ * usbSetInterrupt() and usbSetInterrupt3() are omitted. This is useful if
+ * you need the interrupt-in endpoints in order to comply to an interface
+ * (e.g. HID), but never want to send any data. This option saves a couple
+ * of bytes in flash memory and the transmit buffers in RAM.
+ */
+#define USB_CFG_INTR_POLL_INTERVAL      1
+/* If you compile a version with endpoint 1 (interrupt-in), this is the poll
+ * interval. The value is in milliseconds and must not be less than 10 ms for
+ * low speed devices.
+ */
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/romeo/usbconfig.h]:
```suggestion
```

**Pull request merged**